### ISSUE TITLE
NVDA is not reporting the Chinese symbols input via Chinese input method in some applications

### ISSUE THREAD

**Issue by LittleStar-VIP** [nvaccess/nvda#14842]
### Steps to reproduce:
1. open a blank word file
2. make sure I am using 倉頡 Chinese input
3. using the input method to input '哈佬，你好嗎？我很好。'
4. when i am typing those symbols, nvda is not reporting those symbols '，。？'
5. but if I use up or down arrow to review the line, those symbols are already typed.
A note on how do I enter those symbols,
I have use two methods:
1. pressing ctrl + symbols, e.g. ctrl + , for ，
2. I have also try switching to fullwith typing mode using shift+space and type those symbols

The above problems are not happening in every applications, based on what I have tested so far:
- nvda can correctly report those chinese symbols in windows build notepad application
- nvda cannot report those symbols in word and Google Chrome (the comment text box which I am typing here)

### Expected behavior:
NVDA should report those chinese symbols that I have just typed.

#### NVDA installed/portable/running from source:
portable 
#### NVDA version:
alpha-28111,36a5eae2
#### Windows version:
Windows Home ed 21H2
#### Name and version of other software in use when reproducing the issue:
Microsoft Office 365, Google Chrome 

#### Does the issue still occur after restarting your computer?
not tried
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I just do a quick test and just noticed that nvda 2023.1 stable also have this problem, when I do the test, I have make sure I restart with addons disabled, so it should not be related to addons.

#### If NVDA add-ons are disabled, is your problem still occurring?
no addons are installed
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
not tried

**Comment by Adriani90** [comment]
This depends on which synthesizer you are using. Which one are you using? eSpeak, One core, IBM TTS, Vocalizer etc.? Are only the symbols not reported? Or are you having problems also with the chinese letters?

**Comment by LittleStar-VIP** [comment]
@Adriani90 
When I am doing the test above, I am using Windows OneCore Cantonese Tracy
Those Chinese words are correctly reported.
Well I am not sure if it is something related to the synth, if it is a synth's problem, then it should not be able to report the symbols everywhere, but based on my test, the symbols are reported in notepad.

**Comment by seanbudd** [comment]
Seems related to https://github.com/nvaccess/nvda/issues/14023

**Comment by Adriani90** [comment]
This has very probably been fixed by #16541. Closing as fixed. If you still have this issue on Windows 11, please comment and we can reopen.
